### PR TITLE
Update company-portal-app.md

### PR DESCRIPTION
**Pull request by msbemba**
Added details about who can perform this action

Fixes https://github.com/MicrosoftDocs/memdocs/issues/3713

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: memdocs/intune/apps/company-portal-app.md [hunk 1]
```diff
@@ -43,7 +43,8 @@ The Company Portal apps, Company Portal website, and Intune app on Android are w
 
 ## Customizing the user experience
 
-By customizing the end-user experience, you will help provide a familiar and helpful experience for your end users. To do this, navigate to [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), and select **Tenant Administration** > **Customization**, where you can either edit the default policy or create up to 10 group targeted policies. These settings will apply to the Company Portal apps, Company Portal website, and Intune app on Android.
+By customizing the end-user experience, you will help provide a familiar and helpful experience for your end users. To do this, Sign in as Global Administrator or a 
+Intune Service Administrator and navigate to [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431), and select **Tenant Administration** > **Customization**, where you can either edit the default policy or create up to 10 group targeted policies. These settings will apply to the Company Portal apps, Company Portal website, and Intune app on Android.
```
Comment (JohanFreelancer9) [inline memdocs/intune/apps/company-portal-app.md:47]:
```suggestion
By customizing the end-user experience, you will help to provide a familiar and helpful experience for your end users. To do this, sign in as a Global administrator or an 
Intune service administrator. Navigate to the [Microsoft Intune admin center](https://go.microsoft.com/fwlink/?linkid=2109431) and select **Tenant Administration** > **Customization** where you can either edit the default policy or create up to 10 group targeted policies. These settings will apply to the Company Portal apps, Company Portal website, and Intune app on Android.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by Erikre — APPROVED** [review]

**Comment by Erikre** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**